doc/guide.md

```diff
@@ -3083,6 +3083,35 @@ Note that the call made by Okapi will call auth-token to get a token
 for the tenant this module is enabled for. No user is involved in this.
 Use `modulePermissions` to grant permissions for the token.
 
+With Okapi 4.8.0, the timers can also be specified to occur at given
+times with a specification inspired by cron. This is triggered by suppying
+property `schedule`. This consists of 5 tokens, separated by whitespace.
+The tokens in order are:
+
+ * minute 0-59
+ * hour 0-23
+ * day of month 1-31 *
+ * month 1-12
+ * weekday 1-7, may be given as strings "monday", "tuesday", ..
+
+(*): depending on month and year!
```

Comment:
```suggestion
Time zone is UTC.
```

Reply:
OK. Let's have UTC default. But I think that when we make the timers configurable per-tenant OKAPI-969, we'll have a option to set the locale.